app/src/main/java/com/wireguard/android/services/TaskerIntegrationReceiver.kt

```diff
@@ -17,13 +17,18 @@ import org.koin.core.KoinComponent
 import timber.log.Timber
 
 class TaskerIntegrationReceiver : BroadcastReceiver(), KoinComponent {
```

Comment:
Sorry I'm not 100% sure what you meant by this, could you please elaborate?

Reply:
Move [this](https://github.com/msfjarvis/viscerion/blob/987eeaa18e712205f1d224dd0ff5e485e45ee390/app/src/main/java/com/wireguard/android/services/TaskerIntegrationReceiver.kt#L68) function into the tasker activity and use it for tunnel state changes rather than relying on the broadcast

Reply:
The tasker activity is only used to set the configuration options for the actions to be taken when the action is invoked by Tasker. 

We still need the broadcast receiver class to handle the tunnel state change, as Tasker still sends a specific broadcast when the action is called.

Reply:
> The tasker activity is only used to set the configuration options for the actions to be taken when the action is invoked by Tasker.
> 
> We still need the broadcast receiver class to handle the tunnel state change, as Tasker still sends a specific broadcast when the action is called.

So this PR won't solve anything. Meh, disappointing. Address the other issues and add a CHANGELOG entry.